snap/pack/pack.go

```diff
@@ -92,8 +92,14 @@ func debArchitecture(info *snap.Info) string {
 }
 
 // CheckSkeleton attempts to validate snap data in source directory
-func CheckSkeleton(sourceDir string) error {
-	_, err := loadAndValidate(sourceDir)
+func FCheckSkeleton(w io.Writer, sourceDir string) error {
```

Comment:
I'm not quite sure in this context the "F" prefix brings something, it actually make one think there is a version without, we can change the signature without changing the name.